config/routes.rb

```diff
@@ -1,31 +1,34 @@
 Rails.application.routes.draw do
   root "submissions#new"
-  get "/admin", to: "submission_filters#valid", path: :admin
+
+  get "/submissions", to: "submissions#new"
+  resources :submissions, only: [:new, :create]
+  get "/submissions/thank_you", to: "submissions#thank_you"
+  get "/submissions/closed", to: "submissions#closed"
+  get "/submissions/preparation", to: "submissions#preparation"
 
   devise_for :users, skip: [:passwords, :registrations],
     controllers: { omniauth_callbacks: "users/omniauth_callbacks" }
 
-  get "/submission_filters/valid", to: "submission_filters#valid"
-  get "/submission_filters/rejected", to: "submission_filters#rejected"
-  get "/submission_filters/to_rate", to: "submission_filters#to_rate"
-  get "/submission_filters/results", to: "submission_filters#results"
+  get "/admin", to: "submission_filters#valid", path: :admin
 
-  get "/csv/download_accepted", to: "csv#download_accepted"
-  get "/csv/download_waitlist", to: "csv#download_waitlist"
-  get "/csv/download_unaccepted", to: "csv#download_unaccepted"
+  scope "/admin" do
+    get "/submission_filters/valid", to: "submission_filters#valid"
+    get "/submission_filters/rejected", to: "submission_filters#rejected"
+    get "/submission_filters/to_rate", to: "submission_filters#to_rate"
+    get "/submission_filters/results", to: "submission_filters#results"
 
-  get "/submissions/thank_you", to: "submissions#thank_you"
-  get "/submissions/closed", to: "submissions#closed"
-  get "/submissions/preparation", to: "submissions#preparation"
+    get "/csv/download_accepted", to: "csv#download_accepted"
+    get "/csv/download_waitlist", to: "csv#download_waitlist"
 
-  get "/submissions", to: "submissions#new"
-  resources :submissions, only: [:show, :new, :create] do
-    resource :rate, only: :create
-    resources :comments, only: :create
-  end
+    resources :submissions, only: :show do
+      resource :rate, only: :create
+      resources :comments, only: :create
+    end
 
-  resources :questions, only: [:index, :new, :create, :destroy]
+    get "/settings/", to: "settings#index"
+    put "/settings/update", to: "settings#update"
```

Comment:
Actually I'll leave it for now. It would require some further changes as we're trying make it a singleton not a regular resource.